package.json

```diff
@@ -51,13 +56,21 @@
     "html-webpack-plugin": "^3.2.0",
     "jest": "^24.8.0",
     "mini-css-extract-plugin": "^0.7.0",
+    "moxios": "^0.4.0",
+    "react-redux": "^7.0.3",
```

Comment:
The react-redux, react-router-dom and redux also need to be in dependencies side and not dev dependencies.